Bai2/bai2_2.test.js

```diff
@@ -0,0 +1,12 @@
+const filterArr = require("./bai2_2");
+test("compare arr1 and arr2 return the elements that are present in both arrays", () => {
```

Comment:
The description should indicate the specific function or method being tested and what aspect of it is under examination